### ISSUE TITLE
Help - Setting up a PulseAudio server to connect from Simple Protocol Player app.

### ISSUE THREAD

**Issue by fzadikian** [CypherpunkArmory/UserLAnd#988]
How to use this template:
Any line that begins with '#' is a **header**. Please leave the headers in as is, and fill in the information they suggest
below them. These templates are here to help us resolve your issue as quickly as possible. Leaving requested information out
of your issue will cause our issue template bot to request that your post be reformatted. If a header doesn't seem relevant
to your issue, either write 'N/A' or something similar below the header or consider using a different template.

# Describe the issue you are having.

Basically what's on the title. I tried following some tutorials for UserLand a few months ago but I failed. I also tried recently to connect through either NoMachine or TeamViewer to my Ubuntu XFCE4 fs, with no chance whatsoever. I just want a "simple" way to get audio.

# Describe how your task behaves in a regular Linux environment

I've never set up a PulseAudio server, but I've used vnc, NoMachine and TeamViewer, and as they work on Intel/AMD PCs I thought they would in ARM64. NoMachine claims to work on ARMv8 but I never got it working.

**Comment by FranGamer1892** [comment]
A solution for Debian/LXDE is fine too

**Comment by FranGamer1892** [comment]
😅😁

**Comment by haelmic3** [comment]
I would also be interested in a solution to get pulse audio working... The fact that dbus support is limited or nonexistent gives me less hope that ma solution is available.
I'm not opposed to odd workarounds like needing to use audio port forwarding solution like what must be done on window's subsystem.

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
